repositories/lib/services/entity.repository.js

```diff
@@ -54,6 +54,15 @@ class EntityRepository {
   async update(entity, trx) {
     //Keep the input parameter immutable
     const entityDto = _.cloneDeep(entity);
+
+    const identityClause = {};
+
+    if (_.isArray(this.idColumn)) {
```

Comment:
Would you consider adding test for this case?

Reply:
I am. Gimme a few minutes, I'm working from a web GitHub IDE, just need to upload all the files from the local workstation 😅

Reply:
Hardcore!